### PR TITLE
Exploratory Data Analysis for Pubmed Central and BioRxiv

### PR DESCRIPTION
**Pull request by danich1**
This PR contains results for exploratory data analysis on BioRxiv and Pubmed Central repositories.
Main files to review are:
1. `environment.yml`
2. `EDA_biorxiv.py` -> .ipynb implied
3. `EDA_Pubmed_Central.py` -> .ipynb implied

### CODE REVIEW

File: biorxiv/biorxiv_extractor.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import os
+from pathlib import Path
+import re
+import subprocess
+import tqdm
+
+import pandas as pd
+
+
+files =(
```
Comment (ben-heil) [inline biorxiv/biorxiv_extractor.py]:
```suggestion
files = (
```

File: biorxiv/biorxiv_extractor.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    list(Path("Back_Content").rglob("*.meca"))
+    +
+    list(Path("Current_Content").rglob("*.meca"))
+)
+
+doc_file_hash_mapper = []
+already_seen = set({})
```
Comment (ben-heil) [inline biorxiv/biorxiv_extractor.py]: You don't have to include an empty dict when declaring a set, `already_seen = set()` will do the same thing

File: biorxiv/biorxiv_extractor.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import subprocess
+import tqdm
+
+import pandas as pd
+
+
+files =(
+    list(Path("Back_Content").rglob("*.meca"))
+    +
+    list(Path("Current_Content").rglob("*.meca"))
```
Comment (ben-heil) [inline biorxiv/biorxiv_extractor.py:13]: This is a cool construction, I should use pathlib more than I do

File: biorxiv/exploratory_data_analysis/EDA_BioRxiv.py [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+    p9.ggplot(
+        metadata_df
+        .rename({
+        "bioinformatics":"none",
+        "genomics": "none",
+        "zoology": "none",
+        "evolutionary_biology": "none",
+        "animal behavior and cognition": "none",
+        "ecology":"none"
+        }),
```
Comment (ben-heil) [inline biorxiv/exploratory_data_analysis/EDA_BioRxiv.py:219]: Are article categories something you expect to see in this field? Is it possible that the metadata for the heading is missing, causing the parser to skip the header column? If so, will this disrupt downstream analyses?

**Review by ben-heil — APPROVED** [review]
I left a few minor comments, but everything looks good to me!

File: biorxiv/exploratory_data_analysis/EDA_BioRxiv.py (hunk 4 above)
Comment (danich1) [reply biorxiv/exploratory_data_analysis/EDA_BioRxiv.py:219]: Good catch. When I was analyze this dataset I noted that some of the documents had errors where the document's heading section was replaced with the document categories. I quantified how many documents had this error and it is a negligible amount (15 out of 98k documents). For ease of analysis and maintaining sanity I decided to group these errors into the none category.

**Pull request merged**